instrumentation/grape/lib/opentelemetry/instrumentation/grape/event_handler.rb

```diff
@@ -79,7 +81,7 @@ def handle_payload_exception(span, exception)
             # Only record exceptions if they were not raised (i.e. do not have a status code in Grape)
             # or do not have a 5xx status code. These exceptions are recorded by Rack.
             # See instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/tracer_middleware.rb#L155
-            return unless exception.respond_to?('status') && exception.status.to_i < 500
+            return unless exception.respond_to?('status') && Utils.status_code(exception.status) < 500
```

Comment:
Am I missing something? What is preventing you from using the `Rack::Utils.status_code` method here?

Reply:
nothing, and I am happy to use `Rack::Utils` if it's okay to add `rack` as a dependency.

Reply:
Yes please go ahead. 

Rack is a dependency of this instrumentation already, however in your gemspec please make sure to keep it as a `development` dependency.